README.md

```diff
@@ -337,54 +337,54 @@ You can then use the `Slugify::slugify()` method in your controllers:
 $url = Slugify::slugify("welcome to the homepage");
 ```
 
-### Zend Framework 2
+### Laminas
 
-Slugify can be easely used in Zend Framework 2 applications. Included bridge provides a service and a view helper
+Slugify can be easily used in Laminas applications. Included bridge provides a service and a view helper
 already registered for you.
 
 Just enable the module in your configuration like this.
 
 ```php
-return [
+return array(
     //...
 
-    "modules" => [
-        "Application",
-        "ZfcBase",
-        "Cocur\Slugify\Bridge\ZF2", // <- Add this line
+    'modules' => array(
+        'Application',
+        'Cocur\Slugify\Bridge\Laminas' // <- Add this line
```

Comment:
A note to the automated configuration should be added. Example: https://docs.laminas.dev/laminas-cache/v4/installation/#installation-for-mezzio-and-laminas-mvc-application